EIPS/eip-7291.md

```diff
@@ -0,0 +1,636 @@
+---
+eip: 7291
+title: Purpose bound money
+description: An interface extending ERC-1155 to implement purpose-bound money, a hybrid form of programmable payment and money.
+author: Orchid-Dev (@proj-orchid-straitsx), Victor Liew (@alcedo), Wong Tse Jian (@wongtsejian), Chin Sin Ong (@chinsinong)
+discussions-to: https://ethereum-magicians.org/t/eip-7291-purpose-bound-money/14973
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-24
+requires: 165, 173, 1155
+---
+
+## Abstract
+
+This proposal outlines a smart contract interface that builds upon the [ERC-1155](./eip-1155.md) standard to introduce the concept of a purpose bound money (PBM) defined in the [Project Orchid Whitepaper](../assets/eip-7291/MAS-Project-Orchid.pdf) and [PBM Technical Whitepaper](../assets/eip-7291/PBM-Technical-Whitepaper.pdf).
```

Comment:
An ERC should be able to stand on it's own. We allow linking to resources to support the ERC, but I think these PDFs are too much. PBM should be defined here and it shouldn't refer to Project Orchid at all.

Reply:
Hi  @lightclient, we have make changes to address your comments and a comment on Eth Magician forum. Thanks.